### PR TITLE
This fixes an issue in Traffic Stats with getting the last summary time

### LINKED ISSUE #1826: Traffic Stats error when trying to get LastSummaryTime
When querying Traffic Ops to get LastSummaryTime, Traffic Stats is throwing an error like: 
[ERROR] 2018-01-30 22:33:40 parsing time "2018-01-30 17:31:40-05" as "2006-01-02 15:04:05+00": cannot parse "-05" as "+00".  
Seems like a simple date format change would fix this issue.

### PR DESCRIPTION
**Pull request by dneuman64**
This fixes #1826 

This issue has probably been present in TS since 2.0 when we moved to postgres.

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/951/
Test PASSed.

File: traffic_stats/traffic_stats.go [hunk 1]
```diff
@@ -506,9 +506,9 @@ func getToData(config StartupConfig, init bool, configChan chan RunningConfig) {
 	if err != nil {
 		errHndlr(err, ERROR)
 	} else {
-		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05+00", lastSummaryTimeStr)
+		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05-05", lastSummaryTimeStr)
```
Comment (dangogh) [inline traffic_stats/traffic_stats.go]: shouldn't it be `-07` and not `-05`?   https://gobyexample.com/time-formatting-parsing

Comment (dneuman64) [reply traffic_stats/traffic_stats.go]: hmm...I tested with -05 and that worked, let me try with -07

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/958/
Test PASSed.

**Pull request merged**